extensions/kotlin/runtime/pom.xml

```diff
@@ -56,5 +56,10 @@
             <groupId>org.jetbrains.kotlin</groupId>
             <artifactId>kotlin-stdlib-jdk8</artifactId>
         </dependency>
+        <!-- Is this the right place for this? We don't really want users to have to add it-->
+        <dependency>
+            <groupId>io.smallrye.reactive</groupId>
+            <artifactId>mutiny-kotlin</artifactId>
+        </dependency>
```

Comment:
Are we sure we want to depend on this, or should we opt for Jetbrains':

```xml
    <dependency>
      <groupId>org.jetbrains.kotlinx</groupId>
      <artifactId>kotlinx-coroutines-jdk8</artifactId>
    </dependency>
```

which allows you to await on the `CompletionStage`?

Reply:
I would naively expect we'd want both? :-)

Reply:
Ww don't have to have both because only one would be used under the hood - the implementation would either await for the Uni or for the CompletionStage.

Now if you are talking about whether they should be available for use by the users, then sure, I agree

Reply:
I was thinking about both `Uni` and `CompletionStage` based Java APIs, called from Kotlin `suspend` functions. Kotlin is known for its great Java interop, which facilitates gradual migration and hence mixed language projects. We may call that an advanced use case, but shouldn't neglect it altogether :-)